lang/en/auth_saml2.php

```diff
@@ -47,6 +47,10 @@
 $string['attrsimple_help'] = 'Various IdP\'s such as ADFS use long attribute keys such as urns or namespaced xml schema names. If set to Yes this will simplify these, eg map http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname to such \'givenname\'.';
 $string['certificatedetails'] = 'Certificate details';
 $string['certificatedetailshelp'] = '<h1>SAML2 auto generated public certificate contents</h1><p>The path for the cert is here:</p>';
+$string['checkcertificateexpiry'] = 'SAML certificate expiry';
+$string['checkcertificateexpired'] = 'SAML certificate has expired {$a} ago';
+$string['checkcertificatewarn'] = 'SAML certificate will expire in {$a}';
+$string['checkcertificateok'] = 'SAML certificate will expire in {$a}';
```

Comment:
Referenced in earlier feedback https://github.com/catalyst/moodle-auth_saml2/pull/617/files#r748953445 but linking/mentioning for completeness